sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/FileConfigurationParseTest.java

```diff
@@ -432,7 +453,13 @@ private static java.util.stream.Stream<Arguments> envVarSubstitutionArgs() {
             "key1:\n  ${STR_1}: value1\n",
             mapOf(entry("key1", mapOf(entry("${STR_1}", "value1"))))),
         Arguments.of(
-            "key1:\n - ${STR_1}\n", mapOf(entry("key1", Collections.singletonList("${STR_1}")))));
+            "key1:\n - ${STR_1}\n", mapOf(entry("key1", Collections.singletonList("${STR_1}")))),
+        // Quoted environment variables
+        Arguments.of("key1: \"${HEX}\"\n", mapOf(entry("key1", "0xdeadbeef"))),
+        Arguments.of("key1: \"${STR_1}\"\n", mapOf(entry("key1", "value1"))),
+        Arguments.of("key1: \"${BOOL}\"\n", mapOf(entry("key1", "true"))),
+        Arguments.of("key1: \"${INT}\"\n", mapOf(entry("key1", "1"))),
+        Arguments.of("key1: \"${FLOAT}\"\n", mapOf(entry("key1", "1.1"))));
```

Comment:
It's edge-casey for sure, but for completeness I'd like to see a test where the value is replaced with the empty string. I could see a user down the road wanting to turn something off or just quiet some replacement string with an empty.

Reply:
Good call